Sources/SWBTestSupport/Misc.swift

```diff
@@ -221,8 +221,8 @@ extension ProcessInfo {
                 return p[0] == 1
             }
         }
-        #endif
```

Comment:
This needs a #else branch to continue building on non-Windows

Reply:
Ah, right, Linux won't be able to `import Darwin`.